### PR TITLE
Jenkins package updates (v1.0.0)

### PR DESCRIPTION
**Pull request by rji**
This is the latest version of the Jenkins package in preparation of the 1.0.0 release. It depends on changes in https://github.com/mesosphere/jenkins-mesos/pull/83 that are still being reviewed and tested.

### CODE REVIEW

**Comment by rji** [comment]
@ssk2 — diff of the changes: https://github.com/mesosphere/universe/pull/553/commits/2a947525985af04ef789d3accacb02c7be0562e5

File: repo/packages/J/jenkins/9/resource.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "assets": {
     "container": {
       "docker": {
-        "e84d2658d662": "mesosphere/jenkins:0.2.3"
+        "jenkins-100-16513": "mesosphere/jenkins:1.0.0-1.651.3"
```
Comment (ssk2) [inline repo/packages/J/jenkins/9/resource.json]: I like this versioning scheme!

File: repo/packages/J/jenkins/9/package.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
 {
   "packagingVersion": "2.0",
   "name": "jenkins",
-  "version": "0.2.3",
+  "version": "1.0.0-1.651.3",
   "scm": "https://github.com/mesosphere/jenkins-mesos.git",
   "maintainer": "support@mesosphere.io",
-  "website": "https://jenkins-ci.org",
+  "website": "https://jenkins.io",
```
Comment (ssk2) [inline repo/packages/J/jenkins/9/package.json]: I think it'd be worth updating this website to our docs on dcos.io for Jenkins - what do you think?

Comment (ssk2) [inline repo/packages/J/jenkins/9/package.json]: (Bearing in mind that this is where the docs currently on the jenkins-mesos repo will be moved to.)

File: repo/packages/J/jenkins/9/package.json [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-  "preInstallNotes": "WARNING: Jenkins on DCOS is currently in BETA. There may be bugs, incomplete\nfeatures, incorrect documentation, or other discrepancies.\n\nIf you didn't provide a value for `host-volume` in the CLI,\nYOUR DATA WILL NOT BE SAVED IN ANY WAY.\n",
+  "preInstallNotes": "WARNING: If you didn't provide a value for `host-volume` in the CLI,\nYOUR DATA WILL NOT BE SAVED IN ANY WAY.\n",
   "postInstallNotes": "Jenkins has been installed.",
   "postUninstallNotes": "Jenkins has been uninstalled. Note that any data persisted to a NFS share still exists and will need to be manually removed.",
   "licenses": [
     {
-      "name": "Apache License Version 2.0",
-      "url": "https://github.com/mesosphere/jenkins-mesos/blob/master/LICENSE"
+      "name": "MIT License",
+      "url": "https://jenkins.io/license/"
```
Comment (ssk2) [inline repo/packages/J/jenkins/9/package.json]: This should reflect the license of our integration, which remains Apache 2.0.

Comment (ssk2) [inline repo/packages/J/jenkins/9/package.json]: We can also include multiple licenses here.

**Comment by ssk2** [comment]
Aside from some minor comments, this looks good. Please hold off merging until we discuss what additional changes should go into this package now.

**Comment by rji** [comment]
@ssk2 — ready for review!

File: repo/packages/J/jenkins/9/resource.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-large.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "jenkins-100-16513": "mesosphere/jenkins:1.0.0-1.651.3-rc1"
```
Comment (rji) [inline repo/packages/J/jenkins/9/resource.json]: Oops

**Comment by ssk2** [comment]
@rji: ran the following tests on this and it looks good to me. Merge when you're ready to. 🚀  

+ Basic installation
+ Namespaced + different resource requirements install (e..g jenkins-2)
+ Hostname pinning + task restart
+ Marathon-lb support
+ CD-Demo

**Pull request merged**